lightning/src/chain/chainmonitor.rs

```diff
@@ -738,6 +738,8 @@ where
 			monitor.block_connected(
 				header, txdata, height, &*self.broadcaster, &*self.fee_estimator, &self.logger)
 		});
+		// Assume we may have some new events and wake the event processor
+		self.event_notifier.notify();
```

Comment:
Can you add a comment to the `ChainMonitor::event_notifier` field describing what it's for?